lib/topics/state_and_behaviour.rb

```diff
@@ -2,4 +2,28 @@
 
 # exercise state and behaviour
 module StateAndBehaviour
+  # class to feel yourself real driver
+  class Car
+    attr_accessor :color, :model, :current_speed, :sets, :year
+
+    def self.default_car
+      Car.new {}
+    end
+
+    def initialize(args)
+      raise ArgumentError if args.class != Hash
```

Comment:
`is_a?` or `instance_of?` instead of `!=`. And, please, read about the difference between them. I'll ask it tomorrow 😄